src/en/nineanime/src/eu/kanade/tachiyomi/animeextension/en/nineanime/NineAnime.kt

```diff
@@ -446,8 +446,8 @@ class NineAnime : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
         ListPreference(screen.context).apply {
             key = PREF_DOMAIN_KEY
             title = "Preferred domain (requires app restart)"
-            entries = arrayOf("9anime.to", "9anime.gs", "9anime.pl", "9animehq.to")
-            entryValues = arrayOf("https://9anime.to", "https://9anime.gs", "https://9anime.pl", "https://9animehq.to")
+            entries = arrayOf("9anime.to", "9anime.gs", "9anime.pl", "9anime.id", "9anime.ph", "9animehq.to", "9animeto.io")
+            entryValues = arrayOf("https://9anime.to", "https://9anime.gs", "https://9anime.pl", "https://9anime.id", "https://9anime.ph", "https://9animehq.to", "http://9animeto.io")
```

Comment:
Why not HTTPS for the last one?
Also, their website states that 9animehq.to and 9animeto.io have experimental UI. Does this cause any issues with the extension?

Reply:
Oh I just read your comment.
Then the domains that don't work should be removed completely.

Reply:
> Also 9anime.ph and 9animeto.io are fake websites. Here are all valid ones> https://www.reddit.com/r/9anime/comments/cl5x96/9anime_faqs/

added new domains are of 9anime's pls one's checkout https://9anime.me  
as we know 9anime.me is hub of 9anime listed domains.

Reply:
> Why not HTTPS for the last one? Also, their website states that 9animehq.to and 9animeto.io have experimental UI. Does this cause any issues with the extension?

a small mistake , corrected now.

Reply:
> Oh I just read your comment. Then the domains that don't work should be removed completely.

pls keep all the listed domains , in future the experimental ui can be implemented to all domains i thinks so, just that i think 9anime is testing to improve their website ui, refactoring extension  can be done after they change ui of all their domains

Reply:
> > Oh I just read your comment. Then the domains that don't work should be removed completely.
> 
> pls keep all the listed domains , in future the experimental ui can be implemented to all domains i thinks so, just that i think 9anime is testing to improve their website ui, refactoring extension  can be done after they change ui of all their domains

No we should seriously remove them. We can add them again when the extension actually supports them.

Reply:
sure